diff --git src/google/protobuf/stubs/io_win32.cc src/google/protobuf/stubs/io_win32.cc
index 51656a7..7e1cf78 100644
--- src/google/protobuf/stubs/io_win32.cc
+++ src/google/protobuf/stubs/io_win32.cc
@@ -195,12 +195,12 @@ string normalize(string path) {
   // Join all segments.
   bool first = true;
   std::ostringstream result;
-  for (const auto& s : segments) {
+  for (int i = 0; i < segments.size(); ++i) {
     if (!first) {
       result << '\\';
     }
     first = false;
-    result << s;
+    result << segments[i];
   }
   // Preserve trailing separator if the input contained it.
   if (!path.empty() && is_separator(path[path.size() - 1])) {
