From 04bcf822347e52a4ee1129a87899452bb21cabca Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 16 Sep 2024 11:49:04 +0200
Subject: [PATCH 1/2] Add another aarch64 variant to bzl

---
 python/dist/dist.bzl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/python/dist/dist.bzl b/python/dist/dist.bzl
index 75c21c3..3a89957 100644
--- a/python/dist/dist.bzl
+++ b/python/dist/dist.bzl
@@ -27,6 +27,7 @@ def _get_suffix(limited_api, python_version, cpu):
             "osx-x86_64": "darwin",
             "osx-aarch_64": "darwin",
             "linux-aarch_64": "aarch64-linux-gnu",
+            "aarch64": "aarch64-linux-gnu",
             "linux-x86_64": "x86_64-linux-gnu",
             "k8": "x86_64-linux-gnu",
         }
