From 18f708b5b86905a96e83dbfa1c4c3709a056c6df Mon Sep 17 00:00:00 2001
From: Uwe Korn <uwelk@xhochy.com>
Date: Mon, 16 Sep 2024 11:58:39 +0000
Subject: [PATCH] Support ppc64le

---
 WORKSPACE            | 6 +++---
 python/dist/dist.bzl | 1 +
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index b7bc51c..16087fc 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -25,10 +25,10 @@ py_repositories()
 http_archive(
     name = "platforms",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/platforms/releases/download/0.0.7/platforms-0.0.7.tar.gz",
-        "https://github.com/bazelbuild/platforms/releases/download/0.0.7/platforms-0.0.7.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/platforms/releases/download/0.0.10/platforms-0.0.10.tar.gz",
+        "https://github.com/bazelbuild/platforms/releases/download/0.0.10/platforms-0.0.10.tar.gz",
     ],
-    sha256 = "3a561c99e7bdbe9173aa653fd579fe849f1d8d67395780ab4770b1f381431d51",
+    sha256 = "218efe8ee736d26a3572663b374a253c012b716d8af0c07e842e82f238a0a7ee",
 )
 
 http_archive(
diff --git a/python/dist/dist.bzl b/python/dist/dist.bzl
index 3a89957..cfac558 100644
--- a/python/dist/dist.bzl
+++ b/python/dist/dist.bzl
@@ -28,6 +28,7 @@ def _get_suffix(limited_api, python_version, cpu):
             "osx-aarch_64": "darwin",
             "linux-aarch_64": "aarch64-linux-gnu",
             "aarch64": "aarch64-linux-gnu",
+            "ppc": "powerpc64le-linux-gnu",
             "linux-x86_64": "x86_64-linux-gnu",
             "k8": "x86_64-linux-gnu",
         }
