From b2696c839fc0d69317497a323dda08dc43fe4141 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Tue, 17 Sep 2024 13:58:58 +0200
Subject: [PATCH] Support Windows CPUs

---
 python/dist/dist.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/dist/dist.bzl b/python/dist/dist.bzl
index 87eaf38..7bb4ec3 100644
--- a/python/dist/dist.bzl
+++ b/python/dist/dist.bzl
@@ -5,12 +5,12 @@ load("@system_python//:version.bzl", "SYSTEM_PYTHON_VERSION")
 
 def _get_suffix(limited_api, python_version, cpu):
     """Computes an ABI version tag for an extension module per PEP 3149."""
-    if "win32" in cpu or "win64" in cpu:
+    if "win32" in cpu or "win64" in cpu or "_windows" in cpu:
         if limited_api:
             return ".pyd"
         if "win32" in cpu:
             abi = "win32"
-        elif "win64" in cpu:
+        elif "win64" in cpu or "x64_windows" in cpu:
             abi = "win_amd64"
         else:
             fail("Unsupported CPU: " + cpu)
-- 
2.39.5 (Apple Git-154)

