From 31079cdac726ff90182e38568c5984d2483706b5 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 18 Sep 2024 14:38:37 +0200
Subject: [PATCH] Use Windows visibility

---
 python/protobuf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/protobuf.c b/python/protobuf.c
index bfe776f..681d9ff 100644
--- a/python/protobuf.c
+++ b/python/protobuf.c
@@ -380,7 +380,7 @@ bool PyUpb_IndexToRange(PyObject* index, Py_ssize_t size, Py_ssize_t* i,
 // Module Entry Point
 // -----------------------------------------------------------------------------
 
-__attribute__((visibility("default"))) PyMODINIT_FUNC PyInit__message(void) {
+__declspec(dllexport) PyMODINIT_FUNC PyInit__message(void) {
   PyObject* m = PyModule_Create(&module_def);
   if (!m) return NULL;
 
-- 
2.39.5 (Apple Git-154)

