From 85fe49522b7ba50811d4c0f412873dc9d64eed7a Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sat, 29 Jun 2024 09:49:55 +1100
Subject: [PATCH 2/4] don't mark windows as unsupported

---
 python/BUILD.bazel | 1 -
 1 file changed, 1 deletion(-)

diff --git a/python/BUILD.bazel b/python/BUILD.bazel
index 00238769e..615563465 100644
--- a/python/BUILD.bazel
+++ b/python/BUILD.bazel
@@ -137,7 +137,6 @@ selects.config_setting_group(
 
 # begin:github_only
 _message_target_compatible_with = {
-    "@platforms//os:windows": ["@platforms//:incompatible"],
     "@system_python//:none": ["@platforms//:incompatible"],
     "@system_python//:unsupported": ["@platforms//:incompatible"],
     "//conditions:default": [],
