From 0fcf2f09bb3a5c5f64a012af17cdb2067651cc58 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sat, 29 Jun 2024 09:49:55 +1100
Subject: [PATCH 2/9] don't mark windows as unsupported

---
 python/BUILD.bazel | 1 -
 1 file changed, 1 deletion(-)

diff --git a/python/BUILD.bazel b/python/BUILD.bazel
index 90bfc3dc4..750c89a68 100644
--- a/python/BUILD.bazel
+++ b/python/BUILD.bazel
@@ -103,7 +103,6 @@ selects.config_setting_group(
 )
 
 _message_target_compatible_with = {
-    "@platforms//os:windows": ["@platforms//:incompatible"],
     "@system_python//:none": ["@platforms//:incompatible"],
     "@system_python//:unsupported": ["@platforms//:incompatible"],
     "//conditions:default": [],
