From 2faf3a2758aac4c46c78154bfb9619785f0b0b7d Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 16 Sep 2024 12:59:05 +0200
Subject: [PATCH 2/2] Get python version from environment

---
 python/dist/dist.bzl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/dist/dist.bzl b/python/dist/dist.bzl
index 3a89957..87eaf38 100644
--- a/python/dist/dist.bzl
+++ b/python/dist/dist.bzl
@@ -17,7 +17,7 @@ def _get_suffix(limited_api, python_version, cpu):
         return ".cp{}-{}.{}".format(python_version, abi, "pyd")
 
     if python_version == "system":
-        python_version = SYSTEM_PYTHON_VERSION
+        python_version = SYSTEM_PYTHON_VERSION or os.getenv('PY_VER').replace('.', '')
         if int(python_version) < 38:
             python_version += "m"
         abis = {
