From 44def1d302eab8155f2900d2cd38331b0e56e81f Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sat, 29 Jun 2024 09:49:55 +1100
Subject: [PATCH 2/9] don't mark windows as unsupported

---
 python/BUILD.bazel | 1 -
 1 file changed, 1 deletion(-)

diff --git a/python/BUILD.bazel b/python/BUILD.bazel
index 956e5341e..0d2254394 100644
--- a/python/BUILD.bazel
+++ b/python/BUILD.bazel
@@ -131,7 +131,6 @@ selects.config_setting_group(
 )
 
 _message_target_compatible_with = {
-    "@platforms//os:windows": ["@platforms//:incompatible"],
     "@system_python//:none": ["@platforms//:incompatible"],
     "@system_python//:unsupported": ["@platforms//:incompatible"],
     "//conditions:default": [],
