From 529b68bdeea365559d9c5c6c2ed75401ba4ea089 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 14 Jul 2024 22:44:55 +1100
Subject: [PATCH 4/4] enable windows as a target for python bindings

---
 python/BUILD.bazel | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/python/BUILD.bazel b/python/BUILD.bazel
index f2535de71..712e0726a 100644
--- a/python/BUILD.bazel
+++ b/python/BUILD.bazel
@@ -135,21 +135,11 @@ selects.config_setting_group(
     ],
 )
 
-# begin:github_only
 _message_target_compatible_with = {
-    "@system_python//:none": ["@platforms//:incompatible"],
-    "@system_python//:unsupported": ["@platforms//:incompatible"],
+    "@platforms//os:windows": ["@platforms//:incompatible"],
     "//conditions:default": [],
 }
 
-# end:github_only
-# begin:google_only
-# _message_target_compatible_with = {
-#     "@platforms//os:windows": ["@platforms//:incompatible"],
-#     "//conditions:default": [],
-# }
-# end:google_only
-
 filegroup(
     name = "message_srcs",
     srcs = [
