From 9fc9a1a008bbaf6097976e7f20c9eba29b65935d Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 16 Sep 2024 11:49:04 +0200
Subject: [PATCH 4/9] Support more architectures for Python bindings

---
 python/dist/dist.bzl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/python/dist/dist.bzl b/python/dist/dist.bzl
index 2aca2f1..4ea95a6 100644
--- a/python/dist/dist.bzl
+++ b/python/dist/dist.bzl
@@ -24,7 +24,7 @@ def _get_suffix(ctx, limited_api, python_version):
             return ".pyd"
         if cpu == "x86_32":
             abi = "win32"
-        elif cpu == "x86_64":
+        elif cpu == "x86_64" or "x64_windows" in cpu:
             abi = "win_amd64"
         else:
             fail("Unsupported CPU: " + cpu)
@@ -42,6 +42,8 @@ def _get_suffix(ctx, limited_api, python_version):
                 abi = "x86_64-linux-gnu"
             elif cpu == "aarch64":
                 abi = "aarch64-linux-gnu"
+            elif cpu == "ppc":
+                abi = "powerpc64le-linux-gnu"
             elif cpu == "s390x":
                 abi = "s390x-linux-gnu"
             else:
