From 6e69e5db52bd1dcfff29a78ba07e6c3a3e64cb3b Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 14 Jul 2024 22:44:55 +1100
Subject: [PATCH 4/4] enable windows as a target for python bindings

---
 python/BUILD.bazel | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/python/BUILD.bazel b/python/BUILD.bazel
index 615563465..c4f87f7a8 100644
--- a/python/BUILD.bazel
+++ b/python/BUILD.bazel
@@ -135,21 +135,10 @@ selects.config_setting_group(
     ],
 )
 
-# begin:github_only
 _message_target_compatible_with = {
-    "@system_python//:none": ["@platforms//:incompatible"],
-    "@system_python//:unsupported": ["@platforms//:incompatible"],
     "//conditions:default": [],
 }
 
-# end:github_only
-# begin:google_only
-# _message_target_compatible_with = {
-#     "@platforms//os:windows": ["@platforms//:incompatible"],
-#     "//conditions:default": [],
-# }
-# end:google_only
-
 filegroup(
     name = "message_srcs",
     srcs = [
