From eb82f325f0bf9cd578b5306639256ce84e5cf376 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 16 Sep 2024 11:49:04 +0200
Subject: [PATCH 05/10] Support more architectures for Python bindings

---
 python/dist/dist.bzl | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/python/dist/dist.bzl b/python/dist/dist.bzl
index 7c9095e26..4034e7c9a 100644
--- a/python/dist/dist.bzl
+++ b/python/dist/dist.bzl
@@ -5,12 +5,12 @@ load("@system_python//:version.bzl", "SYSTEM_PYTHON_VERSION")
 
 def _get_suffix(limited_api, python_version, cpu):
     """Computes an ABI version tag for an extension module per PEP 3149."""
-    if "win32" in cpu or "win64" in cpu:
+    if "win32" in cpu or "win64" in cpu or "_windows" in cpu:
         if limited_api:
             return ".pyd"
         if "win32" in cpu:
             abi = "win32"
-        elif "win64" in cpu:
+        elif "win64" in cpu or "x64_windows" in cpu:
             abi = "win_amd64"
         else:
             fail("Unsupported CPU: " + cpu)
@@ -27,6 +27,8 @@ def _get_suffix(limited_api, python_version, cpu):
             "osx-x86_64": "darwin",
             "osx-aarch_64": "darwin",
             "linux-aarch_64": "aarch64-linux-gnu",
+            "aarch64": "aarch64-linux-gnu",
+            "ppc": "powerpc64le-linux-gnu",
             "linux-x86_64": "x86_64-linux-gnu",
             "k8": "x86_64-linux-gnu",
             "s390x": "s390x-linux-gnu",
