From 6558d8f26b9da235ef6eb2e501952efbe839c5cd Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 18 May 2023 19:20:35 +1100
Subject: [PATCH 6/7] explicitly link to abseil_dll on windows

---
 python/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/setup.py b/python/setup.py
index 897191d7b..bdd67a457 100755
--- a/python/setup.py
+++ b/python/setup.py
@@ -338,7 +338,7 @@ if __name__ == '__main__':
               glob.iglob('../third_party/abseil-cpp/absl/**/*.a'))
     else:
       # our windows builds don't follow standard windows naming (i.e. have a lib prefix)
-      libraries = ['protobuf'] if sys.platform != 'win32' else ['libprotobuf']
+      libraries = ['protobuf'] if sys.platform != 'win32' else ['libprotobuf', 'abseil_dll']
       if HasLibraryDirsOpt():
         library_dirs = None
       elif os.path.exists('../bazel-bin/src/google/protobuf/libprotobuf.a'):
