From b99fc3bf8836f93d36b5e6d60f9728092332415a Mon Sep 17 00:00:00 2001
From: pb01ka <gaganpb08singh@gmail.com>
Date: Wed, 27 May 2026 16:38:29 +0530
Subject: [PATCH 11/11] Remove 3.14 from supported python versions to avoid
 duplication error

---
 MODULE.bazel | 1 -
 1 file changed, 1 deletion(-)

diff --git a/MODULE.bazel b/MODULE.bazel
index 89e9bd51b..68f43154d 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -128,7 +128,6 @@ SUPPORTED_PYTHON_VERSIONS = [
     "3.11",
     "3.12",
     "3.13",
-    "3.14",
 ]
 
 # TODO: Replace system_python with hermetic_python.
